chrome/content/zotero/runJS.html

```diff
@@ -1,36 +1,37 @@
 <html id="run-js">
 	<head>
-		<title>Run JavaScript</title>
+		<title data-l10n-id="runJS-title"></title>
 		<link href="chrome://zotero-platform/content/zotero.css" rel="stylesheet"/>
 	</head>
 	<body>
 		<nav>
-			<button onclick="run()">Run</button>
+			<button data-l10n-id="runJS-run" onclick="run()"></button>
 			<span id="run-label"></span>
-			<button id="help-button" onclick="openHelp()">Help</button>
+			<button data-l10n-id="runJS-help" id="help-button" onclick="openHelp()"></button>
 		</nav>
 		
 		<main>
 			<div class="textbox-container">
 				<div class="textbox-header">
-					<label class="textbox-label" for="editor-code">Code:</label>
-					<label id="run-as-async-label" for="run-as-async">
+					<label id="code-editor-label" data-l10n-id="runJS-editor-label" class="textbox-label"></label>
```

Comment:
Revert. "Code:" should be localized, but it is fine as a label. This is a developer tool called "Run JavaScript". It's obvious what "Code" means.

Reply:
Makes sense. In all VPAT issues, the degree to which the labels are descriptive is brought up very often, which is where my "Javascript code editor" came from. But as a rule of thumb, I guess it's better to not change existing labels at this point and we can always revise them later if the VPAT review indicates that it's necessary. I pushed these little tweaks